rajapinnat/logmaster/logmaster-functions.php

```diff
@@ -70,6 +70,18 @@ function logmaster_send_file($filename) {
   }
 }
 
+if (!function_exists('logmaster_sent_timestamp')) {
+  function logmaster_sent_timestamp($tunnus) {
+    global $kukarow;
+
+    $query = "UPDATE lasku SET
+              lahetetty_ulkoiseen_varastoon = now()
+              WHERE yhtio = '{$kukarow['yhtio']}'
+              AND tunnus = '{$tunnus}'";
```

Comment:
Pitää lisätä `AND lahetetty_ulkoiseen_varastoon is null`, että ei päivitetä vahingossa uudestaan.